test/Container/ApplicationFactoryTest.php

```diff
@@ -1071,4 +1071,226 @@ public function testCanSpecifyRouteOptionsViaConfiguration()
         $this->assertInstanceOf(Route::class, $route);
         $this->assertEquals($expected, $route->getOptions());
     }
+
+    /**
+     * @expectedException \Zend\Expressive\Container\Exception\InvalidArgumentException
```

Comment:
Ok, I'll deal these exception messages more detailed after I sync with #195. Thanks for the tip.